dwitter/templates/feed/feed.html

```diff
@@ -36,6 +36,7 @@
   {%elif feed_type == "user" %}
   <li><a class="new" href="{% url 'user_sort_feed' sort='new' url_username=feed_user %}">new</a></li>
   <li><a class="top" href="{% url 'user_sort_feed' sort='top' url_username=feed_user %}">top</a></li>
+  <li><a class="awesome" href="{% url 'user_liked' sort='new' url_username=feed_user %}">liked</a></li>
```

Comment:
I'd prefer to not use "like"-terminology as we don't have a "like" button per se. However, I'm not sure "awesomed" is any better. @sigvef thoughts?